### PR TITLE
Check if user has enrollments

### PR DESCRIPTION
**Pull request by JoshuaKlassen**
Fixes DE: https://rally1.rallydev.com/#/95185052972/detail/defect/111243600552

### CODE REVIEW

File: d2l-user-tile-auto.html [hunk 1]
```diff
@@ -130,7 +130,7 @@
 		_onEnrollmentsResponse: function(response) {
 			if (response.detail.status === 200) {
 				var enrollmentsResponse = window.D2L.Hypermedia.Siren.Parse(response.detail.xhr.response);
-				if (enrollmentsResponse.entities.length === 1) {
+				if (enrollmentsResponse.entities && enrollmentsResponse.entities.length >= 1) {
```
Comment (jstefaniuk-d2l) [inline d2l-user-tile-auto.html]: We still want it to be `=== 1`

**Comment by jstefaniuk-d2l** [comment]
@trevorjonesd2l just a reminder that you'll have to mirror this change in your new behavior (in case you don't get notifications from this repo)

File: d2l-user-tile-auto.html (hunk 1 above)
Comment (JoshuaKlassen) [inline d2l-user-tile-auto.html]: oh whoops, I just did that to see a background image 😛 will update

**Comment by jstefaniuk-d2l** [comment]
:+1: merge away

File: d2l-user-tile-auto.html [hunk 2]
```diff
@@ -130,7 +130,7 @@
 		_onEnrollmentsResponse: function(response) {
 			if (response.detail.status === 200) {
 				var enrollmentsResponse = window.D2L.Hypermedia.Siren.Parse(response.detail.xhr.response);
-				if (enrollmentsResponse.entities.length === 1) {
+				if (enrollmentsResponse.entities && enrollmentsResponse.entities.length === 1) {
```
Comment (omsmith) [inline d2l-user-tile-auto.html]: Isn't what you _really_ want

```js
var enrollmentEntities = enrollmentsResponse.getSubEntitiesByRel(this.HypermediaRels.userEnrollment);

if (enrollmentEntities.length === 1) {
  this._organizationUrl = enrollmentEntities[0].getLinkByRel(this.HypermediaRels.organization).href;
  this.$.organizationRequest.generateRequest();
}
```

Comment (JoshuaKlassen) [inline d2l-user-tile-auto.html]: thanks!

**Pull request merged**